### PR TITLE
Add new production file for CircleCI Integration

### PR DESCRIPTION
**Pull request by danielsanchezd**
Adapting to new k8s version.

### CODE REVIEW

File: .circleci/k8s.production.yml [hunk 1]
```diff
@@ -1,39 +1,29 @@
...
 kind: Deployment
 metadata:
   name: metis
-  labels:
-    app: metis
-    env: production
-    criticality: med
+  namespace: discover
 spec:
   replicas: 2
```
Comment (DennisSchwartz) [inline .circleci/k8s.production.yml:7]: You changed most of them to 1 replica. Any reason why this one stays at two?

File: .circleci/k8s.production.yml [hunk 2]
```diff
@@ -1,39 +1,29 @@
...
       labels:
         app: metis
-        env: production
-        criticality: med
     spec:
       containers:
-        - name: metis
-          image: registry.repositive.io:5000/metis:${TAG}
-          env:
-            - name: DB_URI
```
Comment (DennisSchwartz) [inline .circleci/k8s.production.yml:22]: It's missing a database connection?

**Review by DennisSchwartz — CHANGES_REQUESTED** [review]

**Comment by danielsanchezd** [comment]
Metis is messed up. We will need to make subsequent changes in most of these repos k8s config files before they're usable (and we might even move to helm charts before that happens). You're right, but at this moment I don't think we need to fix this.

**Comment by DennisSchwartz** [comment]
But then we also don't need to deploy it, right? What's the point if it isn't working doesn't have access to a DB?

**Comment by danielsanchezd** [comment]
I just looked at the code and woah... what a mess. This file doesn't even deploy the service. It needs to be modified, definitely. I'll add the cloudsql to it since I'm on it

**Comment by danielsanchezd** [comment]
Let's just close this one for now, according to @DrWilliamsSteven it's not doing anything right now.

**Pull request closed without merging**